packages/front-end/components/GuidedGetStarted/GuidedGetStarted.tsx

```diff
@@ -11,13 +11,13 @@ import { useUser } from "@/services/UserContext";
 import { useDefinitions } from "@/services/DefinitionsContext";
```

Comment:
It doesn't look like we're using this anymore - the code in `getstarted.tsx` has a hard-coded value that means this won't render anymore.